src/aoai-simulated-api/src/aoai_simulated_api/models.py

```diff
@@ -149,6 +150,11 @@ class OpenAIDeployment:
     tokens_per_minute: int
 
 
+@dataclass
+class OpenAIEmbeddingDeployment:
+    name: str
+    size: int
```

Comment:
Would it be possible to combine `size` into the existing `OpenAIDeployment` class as an `embedding_size` property?

I think that would simplify use of the simulator as there would be a single config for the deployments that includes the token per minute and embedding size.

Reply:
Fixed this with latest commit